bin/ci

```diff
@@ -70,6 +70,7 @@ prepare_system() {
   on_linux sudo service docker restart
 
   on_osx brew update
+  on_osx brew update
```

Comment:
Why 2 times?

Reply:
I'm debugging why llvm@8 is not appearing in CircleCI.